src/Controllers/PushController.js

```diff
@@ -55,6 +55,26 @@ export class PushController {
       onPushStatusSaved(pushStatus.objectId);
       return badgeUpdate();
     }).then(() => {
+      // Update audience lastUsed and timesUsed
+      if (body.data && body.data.audience_id) {
+        const audienceId = body.data.audience_id;
+        // Remove the audience_id from the push body.
+        delete body.data.audience_id;
+
+        var updateAudience = {
+          lastUsed: { __type: "Date", iso: new Date().toISOString() },
+          timesUsed: { __op: "Increment", "amount": 1 }
+        };
+        const audienceRestQuery = new RestQuery(config, master(config), '_Audience', {objectId: audienceId});
+        return audienceRestQuery.buildRestWhere().then(() => {
```

Comment:
Let’s not wait on that promise before sending the push? Because increments are atomic, we should be good